Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -33,8 +33,10 @@ public class ProtoTest
             var protoPaths = protoFilenames.Select(x => Path.Combine(Invoker.GeneratorTestsDir, x));
             using (var desc = Invoker.TempFile())
             {
-                Invoker.Protoc($"-o {desc} --experimental_allow_proto3_optional --include_imports --include_source_info " +
```

Comment:
Revert this change? (It doesn't seem to do anything.) If we don't revert it, please outdent the second line of the `protocArgs` declaration.